app/src/main/java/com/example/hepiplant/PlantEditActivity.java

```diff
@@ -396,7 +432,7 @@ private JSONObject makeScheduleJSON() {
         JSONObject scheduleJ = new JSONObject();
 
         try {
-            scheduleJ.put("id",getIntent().getExtras().getLong("scheduleId"));
+            scheduleJ.put("id", plant.getSchedule().getId());
```

Comment:
Czy możesz id zmienić na postId (czy inny obiekt jeśli odnosi się to do czegoś innego)?